code/modules/client/client_procs.dm

```diff
@@ -536,6 +536,10 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	return ..()
 
 /client/Destroy()
+	gc_destroyed = world.time
```

Comment:
why here and not Del?

Reply:
I'll move it

Reply:
Actually how much does this matter, should we call this before the send signal?

![image](https://github.com/tgstation/tgstation/assets/35135081/c1a4fb0c-fe3e-4f74-8367-618e32e59b63)

Reply:
... yeah, I think we should. that's how qdel() does it at least